skore/tests/unit/sklearn/plot/roc_curve/test_comparison_cross_validation.py

```diff
@@ -0,0 +1,125 @@
+"""Test the `roc_curve` display method."""
```

Comment:
I think that we should also add some tests for the kwargs.

I saw potentially a bug and if we have a test for it, I would be more confident.